pkg/templates/aws/edges/rest_api-api_integration.yaml

```diff
@@ -2,3 +2,9 @@ source: aws:rest_api
 target: aws:api_integration
 direct_edge_only: true
 deployment_order_reversed: true
+operational_rules:
+  - steps:
```

Comment:
Probably worth a comment that this is kind of a workaround, maybe link to a ticket.

Reply:
👍